File: SurveyAPI/SurveyAPI/Models/QuestionList.cs
namespace SurveyAPI.Models {
    public class QuestionList {
        public Guid Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public DateTime BeginDate { get; set; } = DateTime.Now;
        public DateTime EndDate { get; set; }
    }
}

File: front-vue/src/models/question/QuestionList.ts
export class QuestionListDTO {
  id = "";
  name = "";
  beginDate = "";
  endDate = "";
}

export class QuestionList extends QuestionListDTO {
  constructor(data: QuestionList | QuestionListDTO | null) {
    super();
    if (data) Object.assign(this, data);
  }
}
